src/sentry/static/sentry/app/views/stream.jsx

```diff
@@ -232,8 +232,15 @@ const Stream = React.createClass({
     let params = this.props.params;
     this.api.request(`/projects/${params.orgId}/${params.projectId}/tags/`, {
       success: tags => {
+        let trimmedTags = tags;
+
+        if (trimmedTags.length > 500) {
+          trimmedTags = trimmedTags.slice(0, 500);
+          Raven.captureMessage('High Tag Number', {level: 'info', tags: tags.length});
```

Comment:
I mean, what are we going to do with this message? If anything, this is missing a lot of information to look into it. I'd more say that we should be messaging this to the user to explain what's happening.

Reply:
mostly i'm curious to track the magnitude of how many people see this

Reply:
messaging like an alert every times they load this page? i'd almost rather do this silently

Reply:
i see how this message isn't super useful but it would get the count and the projects effected

Reply:
> messaging like an alert every times they load this page? i'd almost rather do this silently

Nah, I didn't mean an `alert()`, but I meant some message in the UI. I'm not sure how this looks, but if we're truncating the data, someone might be like, "Why is something I'm looking for missing?" So in my mind like, a red banner on the page that says like, "You have too many tags, so we can render all of these." I dunno.

> i see how this message isn't super useful but it would get the count and the projects effected

Should probably tag then project id or other information that can be used to investigate. :)

Reply:
I like the idea of letting them know that their tags were *truncated* if it makes sense.

Reply:
i meant like alert.jsx which shows at the top of the page